docs/firmware/device-sdk/firmware-upgrade/3-sign-and-flash.md

```diff
@@ -5,12 +5,20 @@ sidebar_position: 2
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+### Signing the sample image
+The following instructions do not apply to Nordic targets.  The build process for the nrf91 includes the signing of images.
```

Comment:
To people less familiar with the Nordic products, this sentence might sounds confusing. I assume all the Nordic targets sign images during the build process, right? Can we say something like "This guide only applies to targets other than Nordic. You don't need to sign images for Nordic targets. The images are automatically signed during the build process"?

Reply:
Sure! I see your point